### DISCUSSION TITLE
Is there a way to load the configuration into the SDDM login screen?

### DISCUSSION THREAD

**Discussion by HolySoap** [sezanzeb/input-remapper#179] [Q&A]
I would like to use the controller as soon as the SDDM shows up.
The distro would be Manjaro MATE on a Raspberry Pi 4.

**Comment by sezanzeb** [comment]
You could use the cli https://github.com/sezanzeb/key-mapper/blob/main/readme/usage.md#cli and use it automatically when your system is booted. https://superuser.com/questions/685471/how-can-i-run-a-command-after-boot

*Just in case* you are also thinking about using key-mapper for entering special characters in your password: I would advise against using passwords that require special tools to enter them.

**Reply by HolySoap** [reply]
Do not worry I use the gamepad mostly as a mouse and use "onboard" as the on-screen keyboard.

I'll try that and report back tomorrow.

**Reply by HolySoap** [reply]
Can't get it to work and I only understand half of it at best, if I login and then logout it works, but I need it right at the start-up before I even log in the first time.
Well seems like I have to go back at fighting with this rubbish 2in1 thingy I have.
